connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -147,6 +148,13 @@ message Expression {
     repeated string target = 1;
   }
 
+  // Represents all of the input attributes to a given relational operator, for example in
+  // "SELECT `(id)?+.+` FROM ...".
+  message UnresolvedRegex {
+    // (Required) The column name used to extract column with regex.
+    string col_name = 1;
```

Comment:
not related to this PR, but I'm thinking we probably need to unify the naming in protos at some moment. Since we are using `col_name`, `column_names`, `cols`, `col1`, `col2` ...